<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSDWidget</class>
 <widget class="QDialog" name="OSDWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>100</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolButton" name="coverArtButton">
     <property name="icon">
      <iconset resource="../../quarkplayer-app/quarkplayer.qrc">
       <normaloff>:/icons/quarkplayer-64x64.png</normaloff>:/icons/quarkplayer-64x64.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="previousTrackButton">
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border: 2px solid gray;
	border-radius: 10px;
	padding: 0 8px;
	background: white;
	selection-background-color: darkgray;
	position: absolute;
	left: 50px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playPauseButton"/>
     </item>
     <item>
      <widget class="QToolButton" name="stopButton"/>
     </item>
     <item>
      <widget class="QToolButton" name="nextTrackButton"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../quarkplayer-app/quarkplayer.qrc"/>
 </resources>
 <connections/>
</ui>
